repo2docker/buildpacks/docker.py

```diff
@@ -18,7 +19,15 @@ def render(self, build_args=None):
         """Render the Dockerfile using by reading it from the source repo"""
```

Comment:
Docstring needs updating, it would also help my review as I'm not confident about anything about a `appendix` relates to.

Reply:
Augmented.